### ISSUE TITLE
[INSTALL] Add several modules to the global-workflow-env/unified-dev module on Orion

### ISSUE THREAD

**Issue by KateFriedman-NOAA** [JCSDA/spack-stack#471]
**Which software in the stack would you like installed?**

The following modules are currently available in a testing spack-stack on Orion but are not currently loaded via the `global-workflow-env/unified-dev` module:

```
crtm/2.4.0
g2tmpl/1.10.2
gsi-ncdiag/1.0.0
sfcio/1.4.1
```

Please add these ^ modules to the `global-workflow-env/unified-dev` module, thanks!

I also need gempak but it's not currently provided via spack-stack. Will open a separate issue for that and load it separately for now.

Current modules loaded by the `global-workflow-env/unified-dev` I'm using for testing:

```
Orion-login-1[229] /work/noaa/global/kfriedma/git/develop_spack/modulefiles$ module show global-workflow-env/unified-dev
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   /work2/noaa/da/dheinzel-new/spack-stack-unified-env-io-updates/envs/unified-dev-test2/install/modulefiles/intel-oneapi-mpi/2021.5.1/intel/2022.0.2/global-workflow-env/unified-dev.lua:
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
whatis("Name : global-workflow-env")
whatis("Version : unified-dev")
whatis("Target : skylake_avx512")
whatis("Short description : Development environment for NOAA's Global Workflow")
help([[Development environment for NOAA's Global Workflow]])
depends_on("bacio/2.4.1")
depends_on("bufr/11.7.1")
depends_on("cdo/2.0.5")
depends_on("esmf/8.3.0b09")
depends_on("g2/3.4.5")
depends_on("grib-util/1.2.3")
depends_on("ip/3.3.3")
depends_on("landsfcutil/2.4.1")
depends_on("met/10.1.1")
depends_on("metplus/4.1.1")
depends_on("ncio/1.1.2")
depends_on("nco/5.0.6")
depends_on("nemsio/2.5.2")
depends_on("nemsiogfs/2.5.3")
depends_on("netcdf-c/4.9.0")
depends_on("netcdf-fortran/4.6.0")
depends_on("prod-util/1.2.2")
depends_on("sigio/2.3.2")
depends_on("sp/2.3.3")
depends_on("ufs-pyenv/1.0.0")
depends_on("w3emc/2.9.2")
depends_on("w3nco/2.4.1")
depends_on("wgrib2/2.0.8")
prepend_path("CMAKE_PREFIX_PATH","/work2/noaa/da/dheinzel-new/spack-stack-unified-env-io-updates/envs/unified-dev-test2/install/intel/2022.0.2/global-workflow-env-unified-dev-rv3kyns/")
prepend_path("CMAKE_PREFIX_PATH","/work2/noaa/da/dheinzel-new/spack-stack-unified-env-io-updates/envs/unified-dev-test2/install/intel/2022.0.2/global-workflow-env-unified-dev-rv3kyns/./")
setenv("PROJ_LIB","/work2/noaa/da/dheinzel-new/spack-stack-unified-env-io-updates/envs/unified-dev-test2/install/intel/2022.0.2/proj-8.1.0-zkdwox6/share/proj")
setenv("global_workflow_env_ROOT","/work2/noaa/da/dheinzel-new/spack-stack-unified-env-io-updates/envs/unified-dev-test2/install/intel/2022.0.2/global-workflow-env-unified-dev-rv3kyns")
```

**Which machines would you like to have the software installed?**

Orion

**Additional context**

I am currently testing out the provided test spack-stack on Orion with the GFS/global-workflow `develop` version (GFSv17+).

**Comment by climbfuji** [comment]
This will be addressed in or as a follow-up to #454

**Comment by KateFriedman-NOAA** [comment]
I have removed `ncl` from the list above. We won't need it for GFSv17.

**Comment by climbfuji** [comment]
> I have removed `ncl` from the list above. We won't need it for GFSv17.

Yay! Thanks.

**Comment by climbfuji** [comment]
I'll make this a follow-up PR to clean up after #454 , because I don't want to go back and change things and then rerun all the tests again.
